src/visitors/inline_visitor.cpp

```diff
@@ -210,25 +211,21 @@ void InlineVisitor::visit_function_call(FunctionCall& node) {
         return;
     }
 
-    auto function_definition = symbol->get_node();
-    if (function_definition == nullptr) {
+    auto nodes = symbol->get_nodes_by_type(
```

Comment:
Since you can have multiple block assigned to one symbol, should you now not iterate through all of them and inline them all? or how do you make sure all gets inlined?

Reply:
If I try to add a procedure and a function with the same name, it is not possible, because when entering a scope (a block) we try to insert a new symbol only if it does not exist.